src/modules/hip/kernel/erase.cpp

```diff
@@ -359,4 +416,138 @@ extern "C" __global__ void erase_pln1_batch_fp32(
 //   {
 //     output[dst_pix_idx] = input[src_pix_idx];
 //   }
-// }
\ No newline at end of file
+// }
+
+
+#if defined(STATIC)
+RppStatus hip_exec_erase_batch(Rpp8u* srcPtr, Rpp8u* dstPtr, Rpp32u* anchor_box_info, Rpp8u* colors, rpp::Handle &handle, Rpp32u* box_offset, RPPTensorFunctionMetaData &tensor_info, Rpp32s in_plnpkdind, Rpp32s out_plnpkdind, Rpp32u max_height, Rpp32u max_width)
+{
+    int localThreads_x = 32;
+    int localThreads_y = 32;
+    int localThreads_z = 1;
+    int globalThreads_x = (max_width + 31) & ~31;
+    int globalThreads_y = (max_height + 31) & ~31;
+    int globalThreads_z = handle.GetBatchSize();
+    InitHandle *handle_obj = handle.GetInitHandle();
+
+    hipLaunchKernelGGL(erase_batch,
+                       dim3(ceil((float)globalThreads_x/localThreads_x), ceil((float)globalThreads_y/localThreads_y), ceil((float)globalThreads_z/localThreads_z)),
+                       dim3(localThreads_x, localThreads_y, localThreads_z),
+                       0,
+                       handle.GetStream(),
+                       srcPtr,
+                       dstPtr,
+                       anchor_box_info,
+                       colors,
+                       box_offset,
+                       handle_obj->mem.mgpu.uintArr[0].uintmem,
+                       handle_obj->mem.mgpu.srcSize.height,
+                       handle_obj->mem.mgpu.srcSize.width,
+                       handle_obj->mem.mgpu.maxSrcSize.width,
+                       handle_obj->mem.mgpu.srcBatchIndex,
+                       handle_obj->mem.mgpu.inc,
+                       handle_obj->mem.mgpu.dstInc,
+                       in_plnpkdind,
+                       out_plnpkdind);
+
+    return RPP_SUCCESS;
+}
+
+RppStatus hip_exec_erase_batch_fp16(Rpp16f *srcPtr, Rpp16f *dstPtr, Rpp32u* anchor_box_info, Rpp16f* colors, rpp::Handle &handle, Rpp32u* box_offset, RPPTensorFunctionMetaData &tensor_info, Rpp32s in_plnpkdind, Rpp32s out_plnpkdind, Rpp32u max_height, Rpp32u max_width)
+{
+//     int localThreads_x = 32;
```

Comment:
Is this intended to be commented out?

Reply:
@rrawther Yes, the f16s alone are still not enabled as we discussed before - this could be a separate PR to enable half precision for HIP